modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java

```diff
@@ -635,20 +639,18 @@ public void testBoostContentsWithMoreVersions() throws Exception {
 			"Article 2.0"
 		).build();
 
+		JournalArticle journalArticle = _journalArticles.get(1);
+
+		journalArticle.setVersion(2.0);
```

Comment:
Why are we setting the version of article #2 to 2? Is this a way to make to make it have more versions? 

Why is Article 2.0 using a different method of having more versions than Article 1.0? And it looks like the case that they would both be considered to have 2 versions.

Reply:
@joshuacords The flake related with this test were because the mathematical formula used to calculate the score was returning a score with a small difference between the articles. We decided to use a higher score to make sure that the element works.

Reply:
To avoid update the element multiple times we used a different way to update the article

Reply:
![Screenshot from 2022-07-21 15-50-52](https://user-images.githubusercontent.com/76820333/180293710-02926147-f6a8-4239-acd4-2567fea65871.png)
![Screenshot from 2022-07-21 15-51-53](https://user-images.githubusercontent.com/76820333/180293717-eaca93eb-c46f-4b72-a71c-74b1c1b1b8d5.png)

Reply:
It could be refactored to use the same way but the version need to be higher to don't flake

Reply:
Other than that we could back to use the default values of the element to have lower scores

Reply:
So the version difference is just a comparison of the version numbers, in this case 1.1 to 2.0, correct? Does our portal even assign version numbers above 1.x?

Reply:
Yep, I tested it yesterday
![Screenshot from 2022-07-22 12-02-58](https://user-images.githubusercontent.com/76820333/180467932-95947b15-1b11-448f-bb29-4c8e5f70dfed.png)